.github/workflows/main.yml

```
@@ -184,6 +223,14 @@ jobs:
         uses: codecov/codecov-action@v3
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+  integration-report:
```

add a empty line for readability
```suggestion
  
  integration-report:
```